sblecdemo/build.gradle

```diff
@@ -36,8 +35,8 @@ dependencies {
 
     api 'com.jakewharton.timber:timber:4.7.1'
 
-    implementation 'com.google.android.material:material:1.1.0-alpha05'
-    implementation 'com.android.support:appcompat-v7:28.0.0'
+    api 'com.google.android.material:material:1.1.0-alpha05'
```

Comment:
Is this really necessary for the public API? There is none because this is the demo project, right? Or am I missing something? If not, please use implementation, because we aways should prefer it for dependency encapsulation and build speed (see https://stackoverflow.com/a/44419574).

Reply:
You're right, `api` is not required here.